content/https/https-6/contents.lr

```diff
@@ -0,0 +1,17 @@
+_model: question
+---
+title: Why Tor Browser doesn't deploy DoH?
+---
+seo_slug: reason-tor-not-deploy-doh
+---
+description:
+There are two main reasons:
+DoH is centralized and there are not many diverse DoH servers today.
+
+It has also its cons:
+•	It adds a central party seeing all Tor Browser users's DNS requests
+•   It might add latency
+•	Several more round-trips across the Tor network for each web request, which would seem to be a huge performance penalty and so on.
+
+Encrypting DNS is good, but DNS was never meant to be centralized, and we should resist that centralization now.

```

Comment:
One of the problems with that argument is that the "encrypted DNS" alternative to DoH is DoT, and that means converting all existing DNS providers to also allow DNS over TLS traffic. That would be great, except that it also means that instead of sending your precious DNS query profile to DoH providers, you send them to your current DNS provider which is, you guessed it, usually Google or your current ISP, both of which probably spy on those requests actively.

So the problem is not necessarily centralization here but who exactly you ask for DNS.